src/Host/Broker/Impl/Sessions/Session.cs

```diff
@@ -129,6 +134,7 @@ public class Session {
                 _pipe = null;
             };
 
+            logger.LogInformation($"Starting RHost process: {rhostExePath} {arguments}");
```

Comment:
Worth mentioning user and session ID here as well. Keep in mind that the log is written as a flat file, so with concurrent incoming requests, the messages will be interleaved. We'll need enough data to figure out which message belongs to which user+session (or just user, if it's something that's not really session-specific, like parsing username or setting up environment).